serverless/serverless-kn-source.adoc

```diff
@@ -0,0 +1,45 @@
+[id="serverless-kn-source"]
+= Using Knative Eventing sources
+include::modules/common-attributes.adoc[]
+include::modules/serverless-document-attributes.adoc[]
+:context: serverless-kn-source
+
+You can use `kn` to list available event sources or event source types for use with Knative Eventing.
+
+For information about installing `kn`, see xref:../knative-client.adoc#knative-client[Installing Knative Client].
+
+[id="serverless-list-source-types_{context}"]
+== Listing available event source types using `kn`
+
+You can list the available event source types in the terminal by using the following command:
+----
+$ kn source list-types
+----
+
+It is also possible to list available event source types in JSON format:
+
+----
+kn source list-types -o yaml
+----
+
+Currently, Knative Eventing supports the following event source types:
+
+* API server event source
+* Sink binding event source
+
+// additional modules TBA for creating and managing these sources
+
+[id="serverless-untyped-sources_{context}"]
+== Using untyped event sources
```

Comment:
I would skip that section for now as we don't have any support in `kn` for untyped, arbitrary sources (would again just write sources instead of event sources).